README.md

```diff
@@ -53,8 +51,8 @@ And then use it:
 1. Fork it!
 2. Create your feature branch: `git checkout -b usa-state-list`
 3. Commit your changes: `git commit -m 'Add some feature'`
-4. Push to the branch: `git push origin usa-state-list`
-5. Submit a pull request :D
+4. Push to the branch: `git -u push origin usa-state-list`
```

Comment:
@anushagadepalli 
Use the standard git push command by changing this line to: `git push -u origin usa-state-list`